src/utils.test.js

```diff
@@ -323,6 +323,46 @@ describe('utils', () => {
     process.env['LUMIGO_VALID_ALIASES'] = undefined;
   });
 
+  test('getTimeoutTimerBuffer -> ENV_VAR', () => {
+    process.env['LUMIGO_TIMEOUT_BUFFER'] = '0.35';
+    expect(utils.getTimeoutTimerBuffer()).toEqual(350);
```

Comment:
how is that possible that it lower than 500? 😂

Reply:
It's from the config, its override our mechanism